lib/myfinance/resources/receivable_account.rb

```diff
@@ -14,12 +14,19 @@ def receive(id, entity_id, params = {})
         end
       end
 
+      def undo_receivement(id, entity_id)
+        http.put(endpoints(id, entity_id, :undo_receivement)) do |response|
+          respond_with_object(response, 'receivable_account')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.